contrib/gocql/gocql/gocql.go

```diff
@@ -56,14 +56,9 @@ func WrapQuery(q *gocql.Query, opts ...WrapOption) *Query {
 		fn(cfg)
 	}
 	if cfg.resourceName == "" {
-		q := `"` + strings.SplitN(q.String(), "\"", 3)[1] + `"`
-		q, err := strconv.Unquote(q)
```

Comment:
I have no idea what this was about: adding quotes only to remove them on the next line...

Reply:
Is there a reason we are parsing the `String()` at all?
It looks like we're just parsing out the 'statement' field. See `String()`: https://github.com/gocql/gocql/blob/master/session.go#L857

Instead of relying on the format of `String()`, can we just use `q.Statement()` to get the statement?

Reply:
Maybe, but that’s a completely separate change I don’t want to get into in this PR. There is are probably considerations there and I know this integrations is heavily used.

Reply:
Looks like it was me who added `q.Statement()`: https://github.com/gocql/gocql/commit/8c2a4423a68d941cd23f221ecf6a0571bb5e7597...

We can't use it because we'll lose support for older versions, where this call was not available.

Reply:
👍 I wasn't sure since gocql isn't versioned.

Reply:
Yeah, even _this_ can break any day.